### PR TITLE
Update radix docs tom

### PR DESCRIPTION
**Pull request by tgberkeley**

### CODE REVIEW

**Comment by Alek99** [comment]
```
─────────────────────────────────────────────────────────────────────────────────────────── Starting Reflex App ────────────────────────────────────────────────────────────────────────────────────────────
Warning: Your version (0.3.6) of reflex is out of date. Upgrade to 0.3.7 with 'pip install reflex --upgrade'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/typer/main.py", line 328, in __call__
    raise e
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/typer/main.py", line 311, in __call__
    return get_command(self)(*args, **kwargs)
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/typer/core.py", line 778, in main
    return _main(
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/typer/core.py", line 216, in _main
    rv = self.invoke(ctx)
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/Users/alekpetuskey/Library/Caches/pypoetry/virtualenvs/reflex-PQ1dIs0F-py3.10/lib/python3.10/site-packages/typer/main.py", line 683, in wrapper
    return callback(**use_params)  # type: ignore
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/reflex/reflex.py", line 250, in run
    _run(env, frontend, backend, frontend_port, backend_port, backend_host, loglevel)
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/reflex/reflex.py", line 181, in _run
    prerequisites.get_compiled_app()
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/reflex/utils/prerequisites.py", line 176, in get_compiled_app
    app_module = get_app(reload=reload)
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/reflex/utils/prerequisites.py", line 160, in get_app
    app = __import__(module, fromlist=(constants.CompileVars.APP,))
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pcweb.py", line 7, in <module>
    from pcweb.pages import blog_routes, doc_routes, routes, faq_routes, changelog_routes
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pages/__init__.py", line 2, in <module>
    from .docs import doc_routes
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pages/docs/__init__.py", line 4, in <module>
    from .components import *
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pages/docs/components/__init__.py", line 4, in <module>
    from pcweb.pages.docs.library import library
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pages/docs/library/__init__.py", line 2, in <module>
    from .typography import *
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pages/docs/library/typography/__init__.py", line 2, in <module>
    from .heading import heading
  File "/Users/alekpetuskey/Desktop/Open Source/pynecone/examples/reflex-web/pcweb/pages/docs/library/typography/heading.py", line 4, in <module>
    heading = component_docpage("docs/library/typography/heading")
TypeError: component_docpage() missing 1 required positional argument: 'comp'
```

Possible problem with the rebase doesn't seem to be running for me anymore

File: docs/library/typography/blockquote.md
[the remark this answers is not in the document]
Comment (tgberkeley) [reply docs/library/typography/blockquote.md:1]: fixed

File: docs/library/typography/blockquote.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Use the `color_scheme` prop to assign a specific color, ignoring the global theme.
+
+
+```python demo
+flex(
+    blockquote("Perfect typography is certainly the most elusive of all arts. Sculpture in stone alone comes near it in obstinacy.", color_scheme="indigo"),
+    blockquote("Perfect typography is certainly the most elusive of all arts. Sculpture in stone alone comes near it in obstinacy.", color_scheme="cyan"),
+    blockquote("Perfect typography is certainly the most elusive of all arts. Sculpture in stone alone comes near it in obstinacy.", color_scheme="crimson"),
+    blockquote("Perfect typography is certainly the most elusive of all arts. Sculpture in stone alone comes near it in obstinacy.", color_scheme="orange"),
+    direction="column",
```
[the remark this answers is not in the document]
Comment (tgberkeley) [reply docs/library/typography/blockquote.md:66]: fixed

File: docs/library/datadisplay/badge.md
[the remark this answers is not in the document]
Comment (tgberkeley) [reply docs/library/datadisplay/badge.md:1]: fixed the styling part

**Review by picklelo — APPROVED** [review]

**Pull request merged**